applets/wncklet/window-list.c

```diff
@@ -997,6 +1015,31 @@ static void setup_dialog_wayland(TasklistData* tasklist)
 }
 #endif /* HAVE_WAYLAND */
 
+static gboolean
+scroll_enabled_mapping_get (GValue   *value,
+                            GVariant *variant,
+                            gpointer  user_data)
+{
+	gboolean active;
+
+	active = g_variant_get_boolean (variant);
+	g_value_set_boolean (value, active);
+
+	return TRUE;
+}
+
+static GVariant *
+scroll_enabled_mapping_set (const GValue       *value,
+                            const GVariantType *expected_type,
+                            gpointer            user_data)
+{
+	gboolean active;
+
+	active = g_value_get_boolean (value);
+
+	return g_variant_new_boolean (active);
+}
```

Comment:
can be removed if you use just `g_settings_bind`

Reply:
Has been updated

Reply:
You can remove `scroll_enabled_mapping_get` and `scroll_enabled_mapping_set`